### DISCUSSION TITLE
Mounted Volume

### DISCUSSION THREAD

**Discussion by futhgar** [jgraph/docker-drawio#122] [Q&A]
Has anyone here tried to add a volume mounted via the compose.yml? Mine keeps failing and I dont see an alternate storage. I'd like to store everything in a separate ZFS share directory instead of the docker volumes..

**Comment by davidjgraph** [comment]
How would you specific that that volume is the locate to save to?

**Reply by WarHawk8080** [reply]
Yes...an environmental variable to be able to save created drawings directly to a volume on the host machine and download/upload would be an EXCELLENT addition!

**Comment by SawkeeReemo** [comment]
Just came here to verify this before I install: So it seems like there's no way to save diagram to a mounted volume on a local machine when hosting this in Docker?

**Comment by davidjgraph** [comment]
That's not what draw.io is designed to do. There no user/auth/file concept on the back-end. There no UI picker to open the file back, etc.
